src/NuGetGallery.Core/GitHub/GitHubUsageConfiguration.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+namespace NuGetGallery.GitHub
+{
+    public class GitHubUsageConfiguration : IGitHubUsageConfiguration
+    {
+        public GitHubUsageConfiguration(IReadOnlyList<RepositoryInformation> reposCache)
+        {
+            NuGetPackagesGitHubDependencies = reposCache.Any()
+                ? GitHubCacheTransformer.GetNuGetPackagesDependents(reposCache)
+                : new Dictionary<string, NuGetPackageGitHubInformation>();
+        }
+
+        public Dictionary<string, NuGetPackageGitHubInformation> NuGetPackagesGitHubDependencies { get; }
```

Comment:
Now that there's `GetPackageInformation`, can this be a private field instead of a public property?